.appveyor.yml

```diff
@@ -2,13 +2,15 @@
 environment:
   nodejs_version: "6"
 
-cache:
-  - node_modules
+#cache:
+# - node_modules
```

Comment:
Yeah, it looks like AppVeyor was just being flaky, rather than the error having anything to do with this PR. I've just uncommented that bit (https://github.com/OctoLinker/browser-extension/pull/286/commits/98010f6e2c7662709e9db3b5f5d37ff7ccf2af59), and it still passes, so I'll go ahead and merge.